spec/client_spec.js

```diff
@@ -36,6 +36,78 @@ describe('Client', function() {
     window.addEventListener.callArgWith(1, evt);
   }
 
+  describe('isDomainValid', function() {
```

Comment:
`isOriginValid` to match the helper method (intent)? Or, possibly just change to:

```js
describe("with a valid origin", () => {})
describe("with an invalid origin", () => {})
```

?